src/main/java/seedu/address/logic/commands/DeleteModuleClassCommand.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.moduleclass.ModuleClass;
+
+public class DeleteModuleClassCommand extends Command {
+
+    public static final String COMMAND_WORD = "delete-class";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the class identified by the index number used in the displayed class list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_DELETE_MODULE_CLASS_SUCCESS = "Delete Class: %1$s";
```

Comment:
Should the tense follow the `DeleteStudentCommand` class instead?
```suggestion
    public static final String MESSAGE_DELETE_MODULE_CLASS_SUCCESS = "Deleted Class: %1$s";
```